docgen/content-sources/v1/toc.yaml

```diff
@@ -135,6 +135,18 @@ toc:
       - title: 'ObjectMetadata'
         path: /docs/reference/functions/providers_storage.objectmetadata.html
 
+  - title: 'functions.tasks'
+    path: /docs/reference/functions/providers_tasks.html
+    section:
+      - title: AuthData
+        path: /docs/reference/functions/common_providers_tasks.authdata.html
+      - title: RateLimits
+        path: /docs/reference/functions/common_providers_tasks.ratelimits.html
+      - title: RetryConfig
+        path: /docs/reference/functions/common_providers_tasks.retryconfig.html
+      - title: TaskContext
+        path: /docs/reference/functions/common_providers_tasks.taskcontext.html
+
```

Comment:
Looks like `TaskQueueBuilder` is missing

Reply:
Should be back now, thanks!